src/angular-app/languageforge/lexicon/views/bootstrap4/settings.html

```diff
@@ -15,18 +15,18 @@
 
 
     <!--Tab Content-->
-    <div class="tab-content">
+    <div class="tab-content" style="margin-bottom:25px">
         <!--Project Settings-->
```

Comment:
I'm not so keen on inline styles.  Can the styling for tabs become universal across the application?